go.mod

```diff
@@ -1,17 +1,17 @@
 module github.com/friendsofgo/gopherapi
 
 require (
-	github.com/DATA-DOG/go-sqlmock v1.4.1
-	github.com/alicebob/miniredis/v2 v2.11.4
+	github.com/DATA-DOG/go-sqlmock v1.5.0
+	github.com/alicebob/miniredis/v2 v2.13.2
 	github.com/gomodule/redigo v2.0.0+incompatible
-	github.com/gorilla/mux v1.7.0
-	github.com/huandu/go-sqlbuilder v1.7.0
+	github.com/gorilla/mux v1.7.4
+	github.com/huandu/go-sqlbuilder v1.8.0
 	github.com/joho/godotenv v1.3.0
-	github.com/lib/pq v1.3.0
-	github.com/openzipkin/zipkin-go v0.2.0
-	github.com/rafaeljusto/redigomock v2.3.0+incompatible
-	github.com/sirupsen/logrus v1.4.2
-	github.com/stretchr/testify v1.2.2
+	github.com/lib/pq v1.8.0
+	github.com/openzipkin/zipkin-go v0.2.3
+	github.com/rafaeljusto/redigomock v2.4.0+incompatible
+	github.com/sirupsen/logrus v1.6.0
+	github.com/stretchr/testify v1.6.1
 )
 
-go 1.13
+go 1.15
```

Comment:
ok but, update the circleci to accept the new versions on tests and build :D

Reply:
Okay, thanks for the suggestion! 🙏 I'm gonna update the Go versions defined on the CI configuration 🚀